server/services/involvedStaffService.js

```diff
@@ -30,6 +30,7 @@ module.exports = function createReportService({ incidentClient, statementsClient
 
     if (!success) {
       const error = new Error('Contains one or more users with unverified emails')
+      // @ts-ignore
       error.notVerified = notVerified
```

Comment:
Generally supposed to create own subclass of errors if going to stick arbitrary stuff on them